### PR TITLE
CC-23709: Added update cart totals based on shipment selector IG.

### PR DESCRIPTION
**Pull request by sakharova-yuliia**
## PR Description

Ticket:  https://spryker.atlassian.net/browse/CC-23709

### CODE REVIEW

File: _includes/pbc/all/install-features/202212.0/install-glue-api/install-the-cart-glue-api.md [hunk 1]
```diff
@@ -0,0 +1,602 @@
+
+Follow the steps below to install Cart feature API.
+
+## Prerequisites
+
+To start feature integration, overview and install the following features and Glue APIs:
+
+| NAME                   | VERSION          | INTEGRATION GUIDE                                                                                                                                                |
+|------------------------|------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Glue API: Spryker Core | {{page.version}} | [Glue API: Spryker Core feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/glue-api-spryker-core-feature-integration.html) |
```
Comment (ilyakubanov) [inline _includes/pbc/all/install-features/202212.0/install-glue-api/install-the-cart-glue-api.md:10]:
```suggestion
| Glue API: Spryker Core | {{site.version}} | [Glue API: Spryker Core feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/glue-api-spryker-core-feature-integration.html) |
```
as in other places, please fix everywhere

File: _includes/pbc/all/install-features/202212.0/install-glue-api/install-the-cart-glue-api.md [hunk 2]
```diff
@@ -0,0 +1,602 @@
...
+| GuestCartItemsByQuoteResourceRelationshipPlugin       | Adds the `guest-cart-items` resource as a relationship to the `guest-carts` resource.                                  |               | Spryker\Glue\CartsRestApi\Plugin\GlueApplication             |
+| CustomerCartsResourceRoutePlugin                      | Configuration for resource routing, how HTTP methods map to controller actions, and if actions are protected.          |               | Spryker\Glue\CartsRestApi\Plugin\ResourceRoute               |
+| CartByRestCheckoutDataResourceRelationshipPlugin      | Adds `carts` resource as relationship by `RestCheckoutDataTransfer.quote`. Applies only for registered customers.      |               | Spryker\Glue\CartsRestApi\Plugin\GlueApplication             |
+| GuestCartByRestCheckoutDataResourceRelationshipPlugin | Adds `guest-carts` resource as the relationship by `RestCheckoutDataTransfer.quote`. Applies only for guest customers. |               | Spryker\Glue\CartsRestApi\Plugin\GlueApplication             |
```
Comment (ilyakubanov) [inline _includes/pbc/all/install-features/202212.0/install-glue-api/install-the-cart-glue-api.md:222]: must be in Checkout as we discussed

File: _includes/pbc/all/install-features/202212.0/install-glue-api/install-the-checkout-glue-api.md [hunk 3]
```diff
@@ -0,0 +1,777 @@
...
+
+Follow the steps below to install Checkout feature API.
+
+## Prerequisites
+
+To start feature integration, overview and install the necessary features:
+
+| FEATURE                                | VERSION          | INTEGRATION GUIDE                                                                                                                                                                        |
+|----------------------------------------|------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Glue API: Spryker Core                 | 202204.0         | [Glue API: Spryker Core feature integration](/docs/scos/dev/feature-integration-guides/202204.0/glue-api/glue-api-spryker-core-feature-integration.html)                                 |
```
Comment (ilyakubanov) [inline _includes/pbc/all/install-features/202212.0/install-glue-api/install-the-checkout-glue-api.md:11]: please fix to {{site.version}} everywhere

**Review by ilyakubanov — CHANGES_REQUESTED** [review]
please create guides where use newly introduced includes

**Review by ilyakubanov — DISMISSED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**